### PR TITLE
add a smart hint helper function to notify user

### PR DESCRIPTION
**Pull request by stardiviner**
This helper function will auto show a message in echo-area when user move point on latex fragement or latex environment.

### CODE REVIEW

**Comment by et2010** [comment]
Thanks! I like your idea on showing status in minibuffer but I'm not very fond of adding functions to `post-command-hook` globally. Maybe the `define-minor-mode` form is a better place to go?

File: org-edit-latex.el [hunk 1]
```diff
@@ -297,6 +297,20 @@ latex-environment."
           (apply oldfun args)))
     (apply oldfun args)))
 
+;;;###autoload
+(defun org-edit-latex-smart-hint ()
+  (if (and (equal major-mode 'org-mode)
+           (or (eq 'latex-environment (car (org-element-context)))
+               (org-inside-LaTeX-fragment-p)))
```
Comment (et2010) [inline org-edit-latex.el]: `org-inside-LaTeX-fragment`, which is designed to be used with cdlatex. returns a non-nil value whenever the cursor is behind a `$`, `\(` or `\[`, even it is not closed yet.  Unlike cdlatex, `org-edit-latex` depends on `org-element` api heavily.

In short, this line could cause an issue. On the other hand, line 303 looks good.

**Review by et2010 — CHANGES_REQUESTED** [review]

File: org-edit-latex.el (hunk 1 above)
Comment (stardiviner) [reply org-edit-latex.el]: You're right, a minor mode is better. I will update my code. Thanks your review.

Comment (stardiviner) [reply org-edit-latex.el]: BTW, can `org-edit-latex` support this inline style LaTeX fragment? Hope can add support for this.

**Comment by stardiviner** [comment]
I updated my code, please review it. Thanks.

**Review by et2010 — APPROVED** [review]

**Comment by et2010** [comment]
Merged. Thanks!

**Pull request merged**